src/definitions/modules/dropdown.js

```diff
@@ -1975,6 +1975,9 @@ $.fn.dropdown = function(parameters) {
                     value    = ( $option.attr('value') !== undefined )
                       ? $option.attr('value')
                       : name,
+                    text     = ( $option.data('text') !== undefined )
```

Comment:
Key `'data'` should be defined in `metadata` so one can configure it as other data attribute do so.

Reply:
It indeed already exists
https://github.com/fomantic/Fomantic-UI/blob/2f3312f0f075e3a570dc96725be17e965b3b0964/src/definitions/modules/dropdown.js#L3989

Reply:
Indeed, should be okay now !